### PR TITLE
Feature/public profile bug

### PR DESCRIPTION
**Pull request by RusAlex**

### CODE REVIEW

File: test-helpers/expect.js [hunk 1]
```diff
@@ -20,4 +20,13 @@ expect.addAssertion('to redirect', function (expect, subject, value) {
   });
 });
 
+expect.addAssertion('to be 200', function (expect, subject, value) {
```
Comment (indeyets) [inline test-helpers/expect.js]: let's find some human-friendlier term? `url to open successfully` maybe?

Comment (RusAlex) [inline test-helpers/expect.js]: 'url to be ok' ?

File: test/integration/anonym-access.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      // await bookshelf.knex.raw('BEGIN;');
+
+    });
+
+    afterEach(async function () {
+      await bookshelf.knex.raw('DELETE FROM users WHERE username=\'john\';');
+    });
+
+    it('Url /user/john works', async function () {
+      return expect(`/api/v1/user/john`, 'to be 200');
```
Comment (indeyets) [inline test/integration/anonym-access.js]: test shouldnb't be about API, but about page

File: test-helpers/expect.js (hunk 1 above)
Comment (indeyets) [inline test-helpers/expect.js]: nope. `url to be ok` means that URL-string is valid. that's not what we test

File: test/integration/anonym-access.js (hunk 2 above)
Comment (RusAlex) [inline test/integration/anonym-access.js]:
```javascript
expect('/api/v1/user/john', 'to open successfully');
```

**Comment by indeyets** [comment]
superseded by #44

**Pull request closed without merging**